package.json

```diff
@@ -36,8 +36,8 @@
     "lib/**/*"
   ],
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "^16.8.6 || ^17",
```

Comment:
I think its fine to just update these to 17.0.1

Reply:
updated :-)